app/lib/dttp/code_sets/institutions.rb

```diff
@@ -194,7 +194,7 @@ module Institutions
         "Point Blank Music School" => { entity_id: "b73e182c-1425-ec11-b6e6-000d3adf095a", hesa_code: "309" },
         "Prestolee SCITT" => { entity_id: "027bf34a-2887-e711-80d8-005056ac45bb" },
         "Queen Margaret University, Edinburgh" => { entity_id: "40f3791d-7042-e811-80ff-3863bb3640b8", hesa_code: "100" },
-        "Queen Mary University of London" => { entity_id: "47f3791d-7042-e811-80ff-3863bb3640b8", hesa_code: "139" },
+        "Queen Mary University of London" => { entity_id: "b93e182c-1425-ec11-b6e6-000d3adf095a", hesa_code: "139" },
```

Comment:
Wondering if we would need to trigger an update to DTTP for the affected records?